AGGREGATE.md

```diff
@@ -185,6 +182,30 @@ true_agg_campaign_counts = raw_agg_campaign_counts / (L1 / 2)
 true_agg_geo_value = 1024 * raw_agg_geo_value / (L1 / 2)
 ```
 
+Note: The `filters` field will still apply to aggregatable reports, and each
+dict in `aggregatable_trigger_data` can still optionally have filters applied
+to it just like for event-level reports.
+
+Note: The `aggregatable_values` field may also be specified as a list of dict,
+where each dict contains a dict of key-value pair as outlined above as well as
+an optional `filters` (and its corresponding `not_filters`) field, allowing
+trigger registrations to customize how values are contributed to keys depending
+on the context of the source.
```

Comment:
Please also mention that if multiple values match, then the first matching one will be used.